homeassistant/components/luxtronik/services.yaml

```diff
@@ -0,0 +1,5 @@
+write:
+  description: Write a parameter on the luxtronik heatpump.
+  fields:
+    parameter: {description: ID of the value to write., example: "ID_Ba_Bw_akt"}
+    value: {description: Value to write., example: "Automatic"}
```

Comment:
Please add an example

Reply:
Can you give point out another integration that shwos how to define an example the right way?
Is [this](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/device_tracker/services.yaml) the correct way to define examples?

Reply:
Yes that is the correct way